framework/components/ecl-profile-topbars/ecl-profile-topbars.scss

```diff
@@ -0,0 +1,31 @@
+/**
+ * ECL Profile Topbar
+ * @define profile-topbar
+ */
+
+.ecl-profile-topbar {
+  margin: 0;
+}
+
+// This is esentially .section__group which can be used when refactored.
+.ecl-profile-topbar__wrapper {
```

Comment:
I think I made it right this time, the result is not totally the same, but hopefully acceptable.